inspire_json_merger/pre_filters.py

```diff
@@ -188,5 +188,28 @@ def _remove_if_falsy(pmap, key):
         return pmap
 
 
+def remove_duplicated_titles(root, head, update):
+    new_elements = []
+    for data in (root, head, update):
+        titles = data.get('titles')
+        if not titles:
+            new_elements.append(data)
+            continue
+        titles_dict = dict()
+        to_delete = []
+        for idx, title in enumerate(titles):
+            if title['title'] not in titles_dict:
+                titles_dict[title['title']] = title
```

Comment:
you're not taking `subtitle` into account.

Reply:
Ah, we should also care about subtitles... argh...

Reply:
:+1: Subtitles support added